spec/smartcar/e2e/vehicle_spec.rb

```diff
@@ -174,6 +174,38 @@
       end
     end
 
+    describe '#batch - success' do
+      context 'with valid and invalid attributes' do
+        it 'should return hash of objects with attribute requested as keys' do
+          expected_description = 'Your application has insufficient permissions to access the requested resource.'\
+          ' Please prompt the user to re-authenticate using Smartcar Connect.'
+          attributes = ['/charge', '/battery', '/odometer', '/fuel']
+          result = @vehicle.batch(attributes)
+
+          expect { result.fuel }.to(raise_error do |error|
```

Comment:
Should this `expect` be after `          expect(result.is_a?(OpenStruct)).to eq(true)`?